### DISCUSSION TITLE
Tem algum repositorio com grade de estudo para programacao

### DISCUSSION THREAD

**Discussion by universe-aung-aung** [universe-aung-aung/earn2#8] [Q&A] [answered]
Eu vi que tem muitas tecnologias na area de desenvolvimento e gostaria de saber se tem alguma api ou alguem tem uma grade do que eu posso estudar?

**Comment by Hein-Htike-Aung** [comment, marked as answer]
Existe sim, este repositorio: tem grade de ensino. <https://github.com/kamranahmedse/developer-roadmap>
